tooling/src/hypothesistooling/projects/hypothesisruby.py

```diff
@@ -57,6 +60,13 @@ def update_changelog_and_version():
     version, version_info = rm.bump_version_info(version_info, release_type)
 
     rm.replace_assignment(GEMSPEC_FILE, 's.version', repr(version))
+
+    # Update to use latest version of conjecture-rust.
+    rm.replace_assignment(
+        CARGO_FILE, 'conjecture',
+        rm.extract_assignment(cr.CARGO_FILE, 'version')
```

Comment:
You’re using `cr.CARGO_FILE` and the `CARGO_FILE` defined in this file interchangeably. Pick one!

Reply:
No I'm not. `cr.CARGO_FILE` is the `conjecture-rust` CARGO_FILE, `CARGO_FILE` is the `hypothesis-ruby` one.

Reply:
\*squints*

Oh yes, you’re right, sorry. That‘s confusing, but I can’t think of a better naming scheme.

Reply:
Maybe the thing to do is to import `cr.CARGO_FILE`  as `CONJECTURE_CARGO_FILE` or something?

Reply:
I think that would be useful, yes.